tests/e2e/CountHistory.test.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect, Page } from "@playwright/test";
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto("count_with_history");
+});
+
+test.describe("Count history page", () => {
+  test("increase count when click `+`", async () => {
+    await page.locator("data-test-id=button-increase").click();
+    const counter = await page.locator("data-test-id=count");
+    await expect(counter).toContainText("1");
+  });
+
+  test("decrease count when click `-`", async () => {
```

Comment:
When I added `test.only` to this test case, it would fail because this test case depends on test case above.
I think we should write test cases which are isolated and can be run independently.

Reply:
@mvn-luanvu-hn 
I wonder e2e must be isolated honestly.
I understand that e2e is the test that functionality works correctly under specific flow.

So, I thought this case, `describe` is the one handles **flows**, and I test the flow of entire history count page that clicks each buttons and see if the result is correct as we expect.

However, I'm not so sure what is the best practice of e2e, so I'll follow what you mentioned if you think all tests are independent.